README.md

```diff
@@ -1,13 +1,15 @@
 # variant-remapping
-Pipeline for remapping VCF variants between two arbitrary assemblies in FASTA format. No chain file is required.
+Pipeline for remapping VCF variants between two arbitrary assemblies in FASTA format. No chain file is required. 
+It does however assume that the source and destination genomes are closely related 
+and was designed with the explicit purpose of lifting over variants from one ersiong of the genome to another.
 
 **Method**: creates reads from the flanking sequences of each variant, then maps them to the new assembly using 
-bowtie2.
+minimap2.
```

Comment:
Does this modification break the bowtie2 workflow?  (It's no problem to stop adding functionality or fixing bugs in the old workflow, but if we actively break it I would prefer removing the code completely or keeping it in a separate branch.)

Reply:
It was, but bowtie2 has an option to support passing the fasta comment as well. 
I've added it so it should now work as well.
This said I'm now happy enough with minmap2 that I don't see the point of keeping bowtie2. 
I would drop it in a subsequent PR so it can be discuss there.

Reply:
makes sense, thank you!

Reply:
> This said I'm now happy enough with minmap2 that I don't see the point of keeping bowtie2.
I would drop it in a subsequent PR so it can be discuss there.

Sure, if you're happy with how minimap operates, then I don't see why we should bother to keep bowtie. So I support dropping it